tensorflow/python/ops/math_ops_test.py

```diff
@@ -372,6 +372,103 @@ def testInvalidOutputTypeMatmul(self):
           self.evaluate(math_ops.matmul(a, b, output_type=dtypes.float32))
 
 
+@test_util.run_all_in_graph_and_eager_modes
```

Comment:
@cantonios Would it be appropriate to remove `@test_util.run_all_in_graph_and_eager_modes` from this test class? I see a handful of other classes without it, but not very many.

Reply:
Not because of a test failure.  It needs to work in all modes.